### ISSUE TITLE
Weird incompatibility between Enigmatic Legacy and Project MMO

### ISSUE THREAD

**Issue by gregest** [Extegral/Enigmatic-Legacy#50]
#### Versions (Be specific, do not write "latest"):
 * Enigmatic Legacy: EnigmaticLegacy-2.0.0.jar
 * Curios API: curious-FORGE-1.16.1-3.0.0.2.jar
 * Patchouli: Patchouli-1.16-39.jar
 * Forge: forge-32.0.108

#### Description of a Bug:
Project MMO's main feature is an RPG style leveling system for a lot of different ascepts in minecraft, such as mining, woodcutting, combat and tools like weapons, pickaxes, axes and everything else have a level requirement to be used properly. Normally if you do not meet the level requirement, then it will act extremely inefficient, tools will be extremely slow meaning even a netherite pickaxe will mine slower than a wooden pickaxe but somehow having both Project MMO and Enigmatic Legacy installed removes this. The game will tell you that you aren't able to use the tool properly, but it will have the speed of normal regardless. 

in short: Enigmatic Legacy disables a feature from Project MMO

#### How to Reproduce:
1. Download both Enigmatic Legacy and Project MMO
2. try to use a tool that you don't meet the level requirement for
3. see that it doesn't function the way it should
  

I also notified Project MMO's dev of the same issue

**Comment by Aizistral** [comment]
I assume this has something to do with the way my mod handles mining speed alteration for items like Charm of Treasure Hunter or Gift of the Heaven. It's already been reported that it breaks mining speed boost from accesories in the Artifact mod, and this issue here probably has the same cause, despite the opposite effect.

I shall attempt to fix it in next patch.
